core/src/main/java/org/frankframework/http/rest/ApiListener.java

```diff
@@ -414,14 +441,16 @@ public void setPrincipalNameClaim(String principalNameClaim) {
 
 	@Override
 	public String toString() {
-		final StringBuilder builder = new StringBuilder();
-		builder.append(getClass().getSimpleName() + "@" + Integer.toHexString(hashCode()));
-		builder.append(" uriPattern["+getUriPattern()+"]");
-		builder.append(" produces["+getProduces()+"]");
-		builder.append(" consumes["+getConsumes()+"]");
-		builder.append(" messageIdHeader["+getMessageIdHeader()+"]");
-		builder.append(" updateEtag["+isUpdateEtag()+"]");
-		return builder.toString();
+		return getClass().getSimpleName() + "@" + Integer.toHexString(hashCode()) +
+				" uriPattern[" + getUriPattern() + "]" +
+				" produces[" + getProduces() + "]" +
+				" consumes[" + getConsumes() + "]" +
+				" messageIdHeader[" + getMessageIdHeader() + "]" +
+				" updateEtag[" + isUpdateEtag() + "]";
```

Comment:
Perhaps that it's me but, when do you prefer to use a StringBuilder vs 'normal' string concatenation?

Reply:
`StringBuilder` when there are conditionals or loops but string concatenation when it is all a single statement.